### PR TITLE
feat: Add starchart command

### PR DESCRIPTION
**Pull request by txtsd**
This should fetch and display the starchart from A plot of the PolyMC repository stars over time

I have no idea what I'm doing so ALL help is appreciated!

### CODE REVIEW

File: src/commands/starchart.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import type { Command } from '../index';
+
+export const cmd: Command = {
+  name: 'starchart',
+  desc: 'Shows a plot of the PolyMC repository stars over time',
+  aliases: ['chart'],
+  exec: async (e) => {
+    const sc_img = await fetch('https://starchart.cc/PolyMC/PolyMC.svg')
+      .then((r) => r as Promise<discord.File(r)>)
+      .then((j) => j);
```
Comment (ryanccn) [inline src/commands/starchart.ts:10]: These two lines do some weird stuff with the types...

`fetch` returns a `Response`, not a `discord.File`

File: src/commands/starchart.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    embed = discord.Embed()
```
Comment (ryanccn) [inline src/commands/starchart.ts:11]: A better way would be to put it all in the message, I think

**Review by ryanccn — CHANGES_REQUESTED** [review]

File: src/commands/starchart.ts (hunk 2 above)
Comment (txtsd) [reply src/commands/starchart.ts:11]: Can you show me a functioning example of it being done? I really can't grok js/ts.

Comment (d-513) [reply src/commands/starchart.ts:11]: discord doesn't display SVG, might be an issue

Comment (txtsd) [reply src/commands/starchart.ts:11]: Damn. Mo' problems.

**Comment by txtsd** [comment]
If anyone wants to push to my branch to make fixes, that's fine too!

**Pull request closed without merging**